contracts/SimpleToken.sol

```diff
@@ -12,7 +12,7 @@ pragma solidity ^0.4.23;
 
 import "./ERC20Token.sol";
 import "./SimpleTokenConfig.sol";
-
+import "./OpsManaged.sol";
```

Comment:
do we need SimpleToken.sol ?

Reply:
No, we do not. 😃 I have removed the same and updated all instances in the tests for MockToken!